NuGet.config

```diff
@@ -7,15 +7,8 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-standard-a5b5f2e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-standard-a5b5f2e1/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-standard-4a9e86e" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-standard-4a9e86e8/nuget/v3/index.json" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
-
-    <!--
-      'src/test/PrepareTestAssets/PrepareTestAssets.proj' generates a NuGet.config file using this
-      one as a template. The following line is a marker to insert the test restore sources.
-    -->
-    <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
```

Comment:
This removal would also cause the PR to fail anyway. Darc bug tracked by https://github.com/dotnet/arcade/issues/3938.